Scripts/AI/BasicAIController.cs

```diff
@@ -25,8 +25,15 @@ public class BasicAIController : MonoBehaviour
         [SerializeField, Tooltip("Log successful behaviour execution debug info to the console.")]
         public bool m_DebugUnsuccessfulToConsole = false;
```

Comment:
For variables like this, if I don't want people changing them at runtime I make them private (SerializeField makes them available in the inspector - not required if public). If I want them to be able to read them then I make a public property with a getter.